pyomo/contrib/appsi/cmodel/src/expression.cpp

```diff
@@ -393,6 +426,12 @@ void MultiplyOperator::evaluate(double* values)
 }
 
 
+void ExternalOperator::evaluate(double* values)
+{
+  assert (false);
+}
+
+

```

Comment:
@michaelbynum Will we ever need this? A comment would be good.

Reply:
Done.